<launch>

 	<node pkg="ORB_SLAM" type="ORB_SLAM" name="ORB_SLAM"  args="Data/ORBvoc.txt Data/Settings.yaml" cwd="node" output="screen">
    <remap from="/orb_slam_keyframes"  to="/svo0/dense_input"/>

   </node>

</launch>
